site/src/routes/docs/index.svelte

```diff
@@ -1 +1,149 @@
-<div>In development</div>
+<script lang="ts">
+</script>
+
+<header class="sticky top-0 text-gray-400 bg-gray-800 body-font ">
+  <div class="flex flex-wrap justify-between items-center p-5 mx-auto md:flex-row">
+    <a class="flex items-center  font-medium   lg:justify-center" href="/docs">
+      <span
+        class="ml-2 text-3xl font-bold text-transparent  bg-clip-text bg-gradient-to-br from-green-400 to-green-600"
+      >
+        Obsidian Docs
+      </span>
+    </a>
+    <div class="inline-flex ml-5 lg:w-2/5 lg:justify-end lg:ml-0">
+      <a
+        class="inline-flex items-center px-3 py-1 font-bold text-white bg-green-500 border-0 rounded-full focus:outline-none hover:bg-green-600"
+        href="https://discord.gg"
+        >Support
+        <svg
+          fill="none"
+          stroke="currentColor"
+          stroke-linecap="round"
+          stroke-linejoin="round"
+          stroke-width="2"
+          class="w-4 h-4 ml-1"
+          viewBox="0 0 24 24"
+        >
+          <path d="M5 12h14M12 5l7 7-7 7" />
+        </svg>
+      </a>
+    </div>
+  </div>
+</header>
+<section class="bg-gradient-to-r from-gray-900 to-gray-800 ">
```

Comment:
This section seems unnecessary.
Also add a main tag with all sections enclosed within it.

Reply:
![image](https://user-images.githubusercontent.com/101550098/163716744-be01c092-e5e8-4a8c-9561-d5b5762b7e96.png)
So should I remove this whole thing?

Reply:
Remove the Documentation heading and Explore our guides... section